ansible-playbook/ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+host_key_checking = False
```

Comment:
Not sure about this being the default. Those with greater ansible-fu than I may wish to opine.

Reply:
I think this should be OK. we may wish to think of a better scheme later